types/tinymce/index.d.ts

```diff
@@ -633,7 +633,7 @@ export interface Formatter {
 
   matchNode(node: html.Node, name: string, vars: {}, similar: boolean): {};
 
-  register(name: {}, format?: {}): void;
+  register(name: {} | string, format?: {} | object[]): void;
```

Comment:
If you think about the things that `{} | string` can fulfill, you have to think about it in terms of the lowest common set of capabilities. In this case, if you're not sure whether you have an empty object or a `string`, you can't guarantee that you have any properties or methods that `string` has, so that's the same as just `{}`.

`{}` is different from `Object` in that if you have something like `{ toString: number }`, that's assignable to `{}` but not `Object`, since `Object` has an explicitly defined `toString` and whatnot.